client/main.lua

```diff
@@ -324,7 +325,29 @@ CreateThread(function()
 
 		if ClosestDoor and ClosestDoor.distance < ClosestDoor.maxDistance then
 			if Config.DrawTextUI and not ClosestDoor.hideUi and ClosestDoor.state ~= showUI then
-				lib.showTextUI(ClosestDoor.state == 0 and lockDoor or unlockDoor)
+			if ClosestDoor.state == 0 then
+			    local text = config.unlockText
+			    local style = {}
+			    if config.useColors then
+				style.backgroundColor = config.unlockColor
+			    end
+			    lib.showTextUI(text, {
+				position = "top-center",
+				icon = 'hand',
+				style = style
+			    })
+			else
+			    local text = config.lockText
+			    local style = {}
+			    if config.useColors then
+				style.backgroundColor = config.lockColor
+			    end
+			    lib.showTextUI(text, {
+				position = "top-center",
+				icon = 'hand',
+				style = style
+			    })
+			end	
```

Comment:
I'm not sure if this is something we'd want but anyways, why not use ternary instead of copy pasting the code twice?

Reply:
```
local text, style = config[ClosestDoor.state == 0 and "unlockText" or "lockText"], {}
if config.useColors then
    style.backgroundColor = config[ClosestDoor.state == 0 and "unlockColor" or "lockColor"]
end
lib.showTextUI(text, {
    position = "top-center",
    icon = 'hand',
    style = style
})
```

Like so?